static/latest-dev-version

```diff
@@ -0,0 +1 @@
+v3.97.0-7-gb348119
```

Comment:
@tgummerer, minor inconsistency with `latest-version`. The `latest-version` doesn't include a `v` prefix. Should we do the same here?

https://www.pulumi.com/latest-version

Reply:
Yes, you're right.  The script also expects us not to have the prefix, I'll fix it in the pulumi/pulumi CI job.